mission-vision-values.md

```diff
@@ -116,3 +116,75 @@ to degrade gracefully as needed.
 OpenTelemetry users should not have to choose between high-quality telemetry and
 a performant application. High performance is a requirement for OpenTelemetry,
 and unexpected interference effects in the host application are unacceptable.
+
+## Community values &mdash; the principles that guide our interactions
+
+The OpenTelemetry project aims to be a welcoming place where new and existing
+members feel safe to respectfully share their opinions and disagreements. We
+want to attract a diverse group of people to collaborate with us, which means
+acknowledging that people come from different backgrounds and cultures. 
+
+There might be situations where community members act in a dubious manner. If
+you have seen or experienced unacceptable behavior or anything that would make
+our community less welcoming, please speak up! See [our code of
+conduct](./code-of-conduct.md) for more information on how to report bad behavior.
+
+While we want to encourage everyone to express themselves in their own way,
+there are some behaviors that we encourage you to adopt while interacting with
+other community members.
+
+#### Act on behalf of the project
+
+It’s no secret that a good number of maintainers of the project are employed by
+companies with commercial interests in OpenTelemetry, especially vendors in the
+observability space. That said, we expect community members to act in the best
+interests of the project. Each member’s priorities can (and should!) align with
+those of their employers so that the relationship is beneficial to all parties,
+but when acting as a maintainer or contributor to the project, community members
+are expected to wear the project’s hat. 
+
+##### Disclose potential conflicts of interest
```

Comment:
do we want to explicitly mention conflicts regarding employers (and/or customers of employers)? the "hats" below are all OTel-specific hats...

Reply:
Can you give me an example of such conflict? I think I see what you mean, but when taking the previous point into consideration ("wear the project's hat"), I'm not sure what else to cover.

I drafted something like this, but wasn't 100% happy:

> a Collector maintainer might be part of the Governance Committee, a JavaScript maintainer might be part of the Technical Committee, a Go approver might have a customer being affected by a specific behavior on the Go's SDK, ...

Reply:
I'm marking this as resolved, as we can tweak this later.